wasm/__init__.py

```diff
@@ -2434,59 +2423,45 @@ def spec_invoke_function_address(config, a=None):
         config["instrstar"] = blockinstrstarendend
         config["idx"] = 0
         config["control_stack"] = []
-        # config_new = {"S":S,"F":F,"instrstar":blockinstrstarendend,"idx":0,"operand_stack":[],"control_stack":[]}
-        # ret = spec_expr(config_new)
-        # if ret=="trap": return ret
```

Comment:
Yikes at all this commented logic. Maybe the `# TODO confirm correct implementation` is warranted here too.